module/Finna/src/Finna/RecordDriver/SolrLido.php

```diff
@@ -1210,7 +1212,7 @@ public function getEvents()
                 'places' => $places,
                 'actors' => $actors,
                 'culture' => $culture,
-                'description' => $description
+                'descriptions' => $descriptions
```

Comment:
Tähän pitäisi jättää myös description-elementti, johon tulisi ensimmäinen description, jos sellainen on. On siis käytössä myös Finnan rajapinnassa, jolloin muuten ei pysy taaksepäin yhteensopivana. Siihen voisi laittaa kommentiksi vaikka, että on `For backward compatibility`.

Reply:
Ok, lisäsin